eng/perf-job.yml

```diff
@@ -0,0 +1,77 @@
+parameters:
+  buildConfig: ''
+  archType: ''
+  osGroup: ''
+  osIdentifier: ''
+  readyToRun: true
+  helixQueues: ''
+  container: ''
+  crossrootfsDir: ''
```

Comment:
You can remove parameters that are not used by this job. For, example `crossrootfsDir`, `helixQueues`, `readyToRun`